charts/openmetadata/templates/ingress.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
There is a list of item that we need to keep as "old k8s versions"?

Kubernetes version lower than 1.22 are not supported anymore (see https://endoflife.date/kubernetes ) and most of the cloud provider do not allow to have it running eg:
 - AWS support 1.20 => 1.22
 - GKE 1.21 => 1.23 https://cloud.google.com/kubernetes-engine/docs/release-notes
 - AKS 1.20 => 1.25 https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#aks-kubernetes-release-calendar

(I'm not try to break previous decision, just understand what are the supported version need to be taken into consideration)

Reply:
Since this Helm Chart is based on `helm create` command. Just want to make it as generic and as provided by Helm Templates for now. 

Regarding "old k8s versions" - 
We will deprecate existing features that we introduce as part of helm charts. Since this is a new implementation, we would like to keep it compatible with any previous versions of Kubernetes.

We would like to support all the kubernetes versions compatible with Helm 3.X.X as of now. Reference - https://helm.sh/docs/topics/version_skew/

Reply:
As per our conversations here, we will need to enable compatibility with kubernetes lower versions as of now that supports Helm v3.

Reply:
We will definitely refactor this for the future releases.